### DISCUSSION TITLE
Commits not being counted as contributions but pull requests do?

### DISCUSSION THREAD

**Discussion by kevinkaw** [community/community#21042] [Other Feature Feedback, Questions, & Ideas] [answered]
I recently started the 100-days-of-code challenge and have been journaling my progress in the log file. I edit this file from the web version of github and any changes I commit to the master branch immediately.

But I notice that this isn't counting towards the contributions. If I submit the changes as a pull request and then merge it does count int the contributions calendar. Can anyone help me on this matter?

I have private contributions and activity overview enabled. Also the forked repository is public.

Thanks in advance!

**Comment by Griphcode** [comment]
Hmm this sounds strange to me. Do you directly edit the files on github or do you edit the manually on your machine?

**Reply by kevinkaw** [reply]
Hello Griphcode, apparently commits to a fork do not count. It needs to be turned into a standalone repository by github support.

**Comment by douglasfdev** [comment]
Do you do all commits on the master branch?
Liste all your branchs:

to see all branches on your PC:
`git branch` 
to see all branches remotes:
`git branch -a`
and see if they are all ok

**Reply by kevinkaw** [reply]
Hi Douglas, apparently commits to a fork do not count. It needs to be turned into a standalone repository by github support. But in any case, concerning your reply. I don't know how to use github desktop yet. So far I'm just editing from the web version.

**Comment by jfullstackdev** [comment, marked as answer]
the repo you are referring to is a fork 

![image](https://user-images.githubusercontent.com/47092464/180433602-abe73540-9468-4060-bd2f-4471330b74d8.png)

all commits will not be counted, 

see this 

https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#commit-was-made-in-a-fork

**Reply by kevinkaw** [reply]
Thanks, I have already contacted github support to turn the fork into a standalone repository.
